kernel/nativevalues.mli

```diff
@@ -368,17 +384,17 @@ val is_parray : t -> bool
 
 val arraymake : t -> t -> t -> t -> t (* accu A n def *)
 val arrayget : t -> t -> t -> t -> t (* accu A t n *)
-val arraydefault : t -> t -> t (* accu A t *)
+val arraydefault : t -> t -> t -> t (* accu A t *)
 val arrayset : t -> t -> t -> t -> t -> t (* accu A t n v *)
 val arraycopy : t -> t -> t -> t (* accu A t *)
 val arraylength : t -> t -> t -> t (* accu A t *)
-val arrayinit : t -> t -> t -> t (* accu A n f def *)
+val arrayinit : t -> t -> t -> t -> t (* accu A n f def *)
```

Comment:
Where does this extra argument come from?

Reply:
Good question. It looks like this has indeed 3 arguments, not 4. I guess I've been confused by the comment, I'll fix this.

Reply:
Fixed. I've set the type of that array conversion functions so that this kind of error cannot occur anymore. Note that this function is actually dead code so this change doesn't matter at all.